### PR TITLE
Generate some JSON data server-side

### PR DESCRIPTION
**Pull request by AlexandreCantin**
SEO engines start to indexing too early and the content we want to be indexed is missing... To avoid that, we make the first request API on the server-side and add it in the window object of JavaScript. So we can put the values immediatly in the Redux store and get the content displayed before AJAX requests.

### CODE REVIEW

**Comment by AlexandreCantin** [comment]
review @vgrange :panama:

**Comment by vgrange** [comment]
relance moi quand c'est ready alors

**Comment by AlexandreCantin** [comment]
review @vgrange

File: frontend/src/components/companies/companies.js [hunk 1]
```diff
@@ -40,22 +39,37 @@ class Companies extends Component {
...
+        // Get longitude,latitude & jobs from window object (and populate server-side)
+        let longitude = parseFloat(this.props.match.params.longitude);
+        if (isNaN(longitude) && window.longitude) longitude = window.longitude;
+
+        let latitude = parseFloat(this.props.match.params.latitude);
+        if (isNaN(latitude) && window.latitude) latitude = window.latitude;
+
+        let defaultJobs = [];
+        if(window.jobs && window.jobs.length > 0) {
+            window.jobs.forEach(job => defaultJobs.push(new Job(job.rome_code, job.label, ''))); // No slug needed)
```
Comment (vgrange) [inline frontend/src/components/companies/companies.js]: `)` en trop

**Pull request merged**